components/accelerator/nvidia/query/nvml/testutil/util.go

```diff
@@ -12,27 +13,69 @@ type mockDevice struct {
 }
 
 func (d *mockDevice) GetArchitectureAsString() (string, error) {
-	return "", nil
+	return "Test Architecture", nil
```

Comment:
can we test this mock? or can we explicitly skip the test coverage check of this file?

Reply:
@xiang90 UTs are added

```
--- PASS: TestCreateDevice (0.00s)
    --- PASS: TestCreateDevice/default_values (0.00s)
    --- PASS: TestCreateDevice/custom_values (0.00s)
PASS
coverage: 97.5% of statements
ok      github.com/leptonai/gpud/components/accelerator/nvidia/query/nvml/testutil      0.258s
```